File: python/Problem007.py
#104743 in 0.25583624839782715 seconds.
import time

from EulerUtils import *

start = time.time()
#nth prime number
target = 10001

count = 0
for i in range(2,999999):
    if is_prime(i):
        count += 1
        if(count == target):
           end = time.time()
           print(i, "in", end-start, "seconds.")
           break

File: java/Problem007.java

public class Problem007 {

	public static void main(String args[]) {
		System.out.println(new Problem007().run());
	}

	public String run() {
		int target = 10001;

		int count = 0;
		for (int i = 2; i < 999999; i++) {
			if (Utils.isPrime(i)) {
				count++;
				if (count == target) {
					return "Answer: " + i;
				}
			}
		}
		return "";
	}
}
